docs/administration-stats.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: administration-stats
 title: Pulsar stats
-sidebar_label: "Pulsar statistics"
```

Comment:
For reviewers: I'll apply the same changes of `administration-stats.md` to 2.11.x and 2.10.x once they are approved by you.